### PR TITLE
✨ Add big text teaser component #1890

### PR DESCRIPTION
**Pull request by fernandolucchesi**
#1890

### CODE REVIEW

File: web/components/src/Text/Text.tsx [hunk 1]
```diff
@@ -60,6 +61,8 @@ export const Text = forwardRef<HTMLDivElement, TextProps>(function Text(
   { size = 'regular', style, children, inverted = false, ...rest },
   ref,
 ) {
+  if (size === 'lg') console.log(ref)
```
Comment (padms) [inline web/components/src/Text/Text.tsx:64]: Remove log here

File: sanityv3/schemas/objects/textBlock.tsx [hunk 2]
```diff
@@ -176,27 +220,27 @@ export default {
...
-              .join('')) ||
-          (ingressBlock &&
-            ingressBlock.children
-              .filter((child: any) => child._type === 'span')
-              .map((span: any) => span.text)
-              .join('')) ||
-          'Missing content!',
-        subtitle: `Text block component.`,
+        title: plainTitle || 'Missing title/content',
+        subtitle: isBigText ? 'Text block component (BIG TEXT)' : 'Text block component',
```
Comment (padms) [inline sanityv3/schemas/objects/textBlock.tsx:243]: ✨🧹✨

File: sanityv3/schemas/objects/teaser.tsx [hunk 3]
```diff
@@ -87,14 +111,35 @@ export default {
...
+      of: [blockContentTypeForBigText],
+      validation: (Rule: Rule) =>
+        Rule.custom((value: PortableTextBlock[], ctx: ValidationContext) => {
+          if ((ctx.parent as Teaser)?.isBigText) {
+            return validateCharCounterEditor(value, 600)
+          }
+          return true
+        }),
+      hidden: ({ parent }: TeaserDocument) => !parent.isBigText,
     },
```
Comment (padms) [inline sanityv3/schemas/objects/teaser.tsx:143]: Can't we use the `Rule.custom((value: PortableTextBlock[]) => validateCharCounterEditor(value, 600)).warning()` and 
`Rule.custom((value: PortableTextBlock[]) => validateCharCounterEditor(value, 600))` for text and bigText.. seems like no difference to me.

Comment (fernandolucchesi) [reply sanityv3/schemas/objects/teaser.tsx:143]: one is a warning, the other is required

**Review by padms — APPROVED** [review]

**Pull request merged**